tests/rptest/services/tls.py

```diff
@@ -316,7 +316,7 @@ def __init__(self, logger, ca_end_date=None, cert_expiry_days=1):
         self._ca_end_date = ca_end_date
         self._cert_expiry_days = cert_expiry_days
         self._ca = self._create_ca()
-        self.certs = {}
+        self.certs: typing.Dict[str, Certificate] = {}
```

Comment:
suggestion: [typing.Dict is deprecated](https://docs.python.org/3/library/typing.html#typing.Dict). this should work:

```suggestion
        self.certs: dict[str, Certificate] = {}
```